tests/acceptance/ManageSubscriptionLinkCest.php

```diff
@@ -62,12 +62,42 @@ public function manageSubscriptionLink(\AcceptanceTester $i) {
     $i->seeNoJSErrors();
   }
 
-  public function unsubscribeLink(\AcceptanceTester $i) {
-    $i->wantTo('Verify that "unsubscribe" link works and subscriber status is set to unsubscribed');
-    $this->sendEmail($i);
+  public function unsubscribeLinksWithLinkTracking(\AcceptanceTester $i) {
+    $i->wantTo('Verify that "unsubscribe" links works with tracking enabled');
+    $this->settings->withTrackingEnabled();
+    $this->verifyUnsubscribeLinks($i);
+  }
 
+  public function unsubscribeLinksWithoutLinkTracking(\AcceptanceTester $i) {
+    $i->wantTo('Verify that "unsubscribe" links works with tracking disabled');
+    $this->settings->withTrackingDisabled();
+    $this->verifyUnsubscribeLinks($i);
+  }
+
+  private function verifyUnsubscribeLinks(\AcceptanceTester $i) {
+    $this->sendEmail($i);
     $formStatusElement = '[data-automation-id="form_status"]';
+    $i->wantTo('Verify that "Unsubscribe List" header link works and subscriber status is set to unsubscribed instantly');
+    $i->amOnMailboxAppPage();
+    $i->click(Locator::contains('span.subject', $this->newsletterTitle));
+    $i->click('#show-headers');
+    $i->waitForText('List-Unsubscribe');
+    $link = $i->grabTextFrom('.headers tbody tr:nth-child(4) td');
```

Comment:
This looks quite flaky. You could select the `td` that is just after `td` with text `List-Unsubscribe` (with XPath it should be possible).

Another option - grab full message source (on the source tab) and parse it out using a regex.